src/service_info.rs

```diff
@@ -1086,6 +1099,47 @@ pub(crate) fn split_sub_domain(domain: &str) -> (&str, Option<&str>) {
     }
 }
 
+/// Represents a resolved service as a plain data struct.
+/// This is from a client (i.e. querier) point of view.
+#[non_exhaustive]
+pub struct ResolvedService {
+    /// service type and domain. For example, "_http._tcp.local."
```

Comment:
Nit: Shouldn't the comments of this structure capitalized? The `sub_ty_domain` is capitalized and even the functions ones

Reply:
Fixed. Thanks!